graylog2-server/src/main/java/org/graylog/plugins/views/search/engine/normalization/PluggableSearchNormalization.java

```diff
@@ -93,6 +102,7 @@ public Query preValidation(final Query query, final ParameterProvider parameterP
         Query normalizedQuery = query;
         if (!query.hasStreams()) {
             normalizedQuery = query.addStreamsToFilter(searchUser.streams().loadMessageStreamsWithFallback());
+        } else if (!query.usedStreamCategories().isEmpty()) {
```

Comment:
@kingzacko1 was this empty clause needed?

Reply:
No I don't think this is actually called anywhere. I started to add the logic there then realized that. I will go ahead and add it just to be consistent.